JavaFXUserInterface/src/main/java/Utilities/TextConstants.java

```diff
@@ -9,6 +15,9 @@
     public static final String WARNING_THRESHOLD = "Warning";
     public static final String FAILED_THRESHOLD = "Failed";
 
+    public static final String ASSETS = "/Assets";
+    public static final String ASSET_TYPE_LIST = "/AssetTypeList";
+    public static final String ALERT_TITLE = "Confirmation Dialog";
```

Comment:
The Dialog windows strings should have something in their name to refer to the type. Such as: ALERT_DIALOG_TITLE

Reply:
Good suggestion. It will be changed